SoftLayer/managers/block.py

```diff
@@ -25,9 +25,10 @@ def list_block_volume_limit(self):
         """
         return self.get_volume_count_limits()
 
-    def list_block_volumes(self, datacenter=None, username=None, storage_type=None, **kwargs):
+    def list_block_volumes(self, datacenter=None, username=None, order=None, storage_type=None, **kwargs):
```

Comment:
the new parameter `order` should go after `storage_type`, not before. This way anyone passing in arguments without the names wont encounter an error.

Reply:
Done.